tests/fixtures/MakeController/config/packages/test/framework.yaml

```diff
@@ -0,0 +1,2 @@
+framework:
+    test: true
```

Comment:
This shouldn't be needed, so we need to dig deeper to see what's going on. 

In 5.3, this file IS gone in the recipe - but it's because it's replaced by the per-env setting: https://github.com/symfony/recipes/blob/9a3759a60b0a5ff48b30f949d2ac89a7d2816e47/symfony/framework-bundle/5.3/config/packages/framework.yaml#L20-L22

So if that isn't working, we need to figure out why. Good job pushing that forward!

Reply:
Alrighty, I've figured out the problem. We had a `framework.yaml` fixture for this particular test that I believe is not needed. This in turn overrides the flex recipe for `framework.yaml` in the generated project. End of the day, `when@test` didn't exist nor did the old `test/framework.yaml` 

All should be fine and well now